src/theme/styled.d.ts

```diff
@@ -62,6 +62,7 @@ export interface Colors {
 
   //blues
   primary: Color
+  primary30: Color
```

Comment:
This is not a good name

Reply:
not sure which is better, can improve later